javav2/example_code/rds/Readme.md

```diff
@@ -2,19 +2,47 @@
 
```

Comment:
Update line 1: 
# Amazon RDS code examples for the SDK for Java

Reply:
fixed